wfc_core/src/lib.rs

```diff
@@ -239,6 +265,19 @@ impl World {
         slot.iter()
     }
 
+    pub fn set_slot_module_weights(&mut self, pos: [u16; 3], weights: &[f32]) {
```

Comment:
Document that this doesn't check validity of the weights and that the caller has to check it.